src/EditorFeatures/CSharpTest/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProviderTests.cs

```diff
@@ -465,5 +465,31 @@ enum Colors
 ";
             await VerifyItemExistsAsync(markup, "Colors");
         }
+
```

Comment:
It preselects provides items based on Enum target typing or the <CompletionList> documentation comment tag